### DISCUSSION TITLE
Github pages. 404 not found, and publish again after unpublishing the page

### DISCUSSION THREAD

**Discussion by JuliaMersing** [community/community#33172] [Other Feature Feedback, Questions, & Ideas]
Hi, I've always use githubPages for publish my works, but with my new repository it was imposible, I always have a 404 page not found. I read and try a lot of different things but still doesn't work, so:
1. I delete de environment GitHub Pages
2. I unpublished my page 
In order to start all over again, but when I go to pages, I can't create a new one or have access to the oldest one. Can someone help me pleas?

I found the same question, but without answer in Stackoverflow https://stackoverflow.com/questions/73593914/publish-again-unpublished-github-pages-project

**Comment by jfullstackdev** [comment]
can you send the direct link of the repo ? so that I can check too

**Reply by JuliaMersing** [reply]
Is resolverd! You can re-publish your GitHub Pages website by navigating to the Actions tab of your repository, click on the Pages build and deployment workflow, then click the Re-run all jobs button located in the upper right corner of the page. After the workflow run has completed, your site will be published.
